sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -446,7 +446,7 @@ and [language and regional support][language_and_regional_support].
 
 ### Healthcare Entities Analysis
 
-[Long-running operation](#long-running-operations) [`begin_analyze_healthcare_entities`][analyze_healthcare_entities] extracts entities recognized within the healthcare domain, and identifies relationships between entities within the input document and links to known sources of information in various well known databases, such as UMLS, CHV, MSH, etc.
+[Long-running operation](#long-running-operations) [begin_analyze_healthcare_entities][analyze_healthcare_entities] extracts entities recognized within the healthcare domain, and identifies relationships between entities within the input document and links to known sources of information in various well known databases, such as UMLS, CHV, MSH, etc.
```

Comment:
this link is taking me here: https://azuresdkdocs.blob.core.windows.net/$web/python/azure-ai-textanalytics/latest/azure.ai.textanalytics.html#azure.ai.textanalytics.TextAnalyticsClient.begin_analyze_healthcare_entities

but it shows the TA client not the healthcare method. Not sure it if is intentional, so pointing out

Reply:
Interesting. It seems that the `latest` link is not quite working... I'll follow up on that

Reply:
yeah the latest link has been buggy at times sadly :'(